visual-recognition/v4.ts

```diff
@@ -118,8 +118,8 @@ class VisualRecognitionV4 extends BaseService {
       }
 
       const formData = {
-        'collection_ids': Array.isArray(_params.collectionIds) ? _params.collectionIds.join(',') : _params.collectionIds,
-        'features': Array.isArray(_params.features) ? _params.features.join(',') : _params.features,
+        'collection_ids': _params.collectionIds,
+        'features': _params.features,
```

Comment:
These need to manually be changed back to normal after generating, I forgot to go over that with you